packages/perseus-editor/src/editor.tsx

```diff
@@ -18,13 +15,11 @@ import * as ReactDOM from "react-dom";
 import _ from "underscore";
 
 import DragTarget from "./components/drag-target";
+import WidgetEditor from "./components/widget-editor";
+import WidgetSelect from "./components/widget-select";
```

Comment:
Just a pure code move and then the removed `import`s here are because they are only used by `WidgetEditor` and `WidgetSelector` and so aren't needed in this file anymore.